docs/installing-vyper.rst

```diff
@@ -76,4 +76,31 @@ Each tagged version of vyper is uploaded to `pypi <https://pypi.org/project/vype
 To install a specific version use:
 ::
 
-    pip install vyper==0.1.0b17
+    pip install vyper==0.2.15
+
+
+nix
+***
+
+Nix is a powerful package manager for Linux and other Unix systems that makes package management reliable and reproducible. 
```

Comment:
Can you please change this section to mirror the other sections? Just say what the platform is and the instructions. I'm ok with including helpful instructions for nix but this is not an appropriate place for marketing nix or nixOS.

Reply:
I actually took this directly from their site as i didnt know what to put. 

Will tone down the commentary.

Thanks,

Sam

Reply:
thanks @charles-cooper for the feedback, I have addressed your comments at [`4914f1a` (#2405)](https://github.com/vyperlang/vyper/pull/2405/commits/4914f1aed2b872328039f02be1f1aaf0c51589bb)

Let me know if you like a rebase / squash to finalize,

Thanks